### PR TITLE
🔨 Refactored README, Installation, Contributing

### PR DESCRIPTION
**Pull request by sschleemilch**
See rendered version: https://github.com/sschleemilch/vss-tools/tree/feature/docs-quickstart

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
 The overall goal of VSS Tools is to provide a set of tools that can be used to convert or verify Vehicle Signal Specifications defined by the format specified by the [COVESA VSS project](https://github.com/COVESA/vehicle_signal_specification).
 VSS Tools is developed in parallel with VSS, please visit the [COVESA VSS project](https://github.com/COVESA/vehicle_signal_specification) for information on how to contribute.
 If any questions arise, please check out the [FAQ](FAQ.md) for more information.
 
-## Available Tools
+## Installation
+
+There are several ways of installing `vss-tools`.
+If you would like to contribute then please follow the [contribute](#contributing) section instead.
+All of them are recommended to be done in an activated python virtual environment:
```
Comment (erikbosch) [inline README.md:14]: As we have seen some people struggling to get a working Python environment I am thinking if we should give some more guidance here, either just add a link to venv documentation like "[activated python virtual environment](https://docs.python.org/3/library/venv.html)" or possibly also show an example how to do it, like in this README

https://github.com/eclipse-kuksa/kuksa-python-sdk/blob/main/docs/building.md

Comment (sschleemilch) [reply README.md:14]: Added instructions how to create a venv

**Review by erikbosch — APPROVED** [review]
LGTM. As this is a minor change I think we can take a merge decision on Tuesday unless someone wants more time to review.

**Comment by erikbosch** [comment]
MoM:

- Merge

**Pull request merged**